aspnetcore/fundamentals/openapi/aspnetcore-openapi.md

```diff
@@ -239,10 +239,14 @@ Create a new ASP.NET Core Web API (Native AOT) project.
 dotnet new webapiaot
 ```
 
+:::moniker-end
+
+:::moniker range="= aspnetcore-9.0"
```

Comment:
Does this only apply to 9, and not 10+ ?

Reply:
Please have a look at the linked issue/PR in the ASP.NET Core repo for more information. 

In .NET 9, you have to add a package and add some code to get OpenAPI to work.

My PR for ASP.NET Core adds this to the AOT template out of the box. Therefore, these steps are only necessary for .NET 9. In .NET 10 and up, you only need to create a new project and you're done.

So, the moniker only shows the steps of adding the package and code for .NET 9.